app/templates/components/select-date-council.hbs

```diff
@@ -25,6 +28,9 @@
       </div>
       <div class="col--6-12">
         <div class="form-item form__row">
+          <label class="vl-form__label">
+            Tot
```

Comment:
Can we make this `Tot en met`? In Kaleidos, that took away a lot of confusion, since "tot" is ambiguous ... ( https://vrttaal.net/taaladvies-taalkwestie/tot-tot-en-met )

Reply:
Sure, I don't know dutch so I just put whatever you tell me :)